radicle-cli/src/commands/path.rs

```diff
@@ -9,7 +9,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "path",
-    description: "Display the radicle home path",
+    description: "Display the home path",
```

Comment:
referring to 'radicle' here may make sense.  The term 'home path' is to close to the more understood user 'home path'

Reply:
Agreed.